selfdrive/controls/plannerd.py

```diff
@@ -37,8 +37,10 @@ def plannerd_thread(sm=None, pm=None):
 
     if sm.updated['modelV2']:
       lateral_planner.update(sm)
+      cloudlog.timestamp("Lateral planner updated")
```

Comment:
move these to before the update. the logMonoTime is basically determined right after this

Reply:
True, but the previous log mono is determined right before if we move.

I think deleting is best, together with solverExecutionTime we have enough info